trade-finance-manager-ui/component-tests/case/facility/facility.component-test.js

```diff
@@ -52,6 +52,24 @@ describe(page, () => {
     wrapper.expectElement('[data-cy="facility-tenor"]').toExist();
   });
 
+  describe('when shouldDisplayFacilityEndDate is false', () => {
+    it('should not render facility end date', () => {
+      wrapper = render({ ...params, shouldDisplayFacilityEndDate: false });
+      wrapper.expectElement('[data-cy="facility-is-using-facility-end-date"]').notToExist();
```

Comment:
same as above

Reply:
See above

Reply:
has this been refactored? if so please resolve